src/twisted/conch/test/test_keys.py

```diff
@@ -877,6 +953,25 @@ def test_fromBlobEd25519(self):
         self.assertTrue(ed25519Key.isPublic())
         self.assertEqual(ed25519PublicData, ed25519Key.data())
 
+    @skipWithoutEd25519
```

Comment:
Do we have test environments without support for ED25519 ?

Reply:
Mh that's something that I wanted to ask, I still see this decorator in the test suite, so I was guessing that maybe still exist a test env that doesn't support ED25519. I think that we can create a follow-up task to remove this if such env doesn't exist.

Reply:
the thing is, nobody took to time to do the cleanup for the existing code.

But I think that is safe to stop using the decorator,
as least for the new code.

Reply:
It makes sense! I removed the decorator here :)